tests/utils.yaml

```diff
@@ -1,7 +1,7 @@
 # Validation tests for utilities
 # see notebooks/appendices/Validation%20Tests.ipynb
 
-get_vmc_sequence_identifier:
```

Comment:
This key is a function name, so needs to be changed in both places.

Reply:
Agreed. These notebooks have been moved to vr-python, though, so I've created a separate issue there to track this.

https://github.com/ga4gh/vr-python/issues/23